app/controllers/application_controller.rb

```diff
@@ -338,7 +338,7 @@ def check_empty_taxonomy
   # If the user has a fact_filter then we need to include :fact_values
   # We do not include most associations unless we are processing a html page
   def included_associations(include = [])
-    include += [:hostgroup, :compute_resource, :operatingsystem, :environment, :model ]
+    include += [:hostgroup, :compute_resource, :operatingsystem, :environment, :model, :reports, :host_statuses ]
```

Comment:
Including :reports here is going to be a really expensive call:

    2015-08-21T12:34:00 [sql] [D]   Report Load (9.7ms)  SELECT "reports".* FROM "reports" WHERE "reports"."host_id" IN (110, 109, 170, 82, 168, 118)

If a host has many reports, it's going to pull back a potentially huge number of records.